chatbot.js

```diff
@@ -93,11 +98,11 @@ function _getBotHelp() {
 function reply_to_user(user, message) {
   const displayName = user.displayName;
   const username = user.username;
-
   if (message.startsWith(BOT_MENTION_NAME)) {
     const parsedMessage = message.slice(BOT_MENTION_NAME.length + 1);
-    console.log(parsedMessage);
     const startsWithString = _getStartsWith(parsedMessage);
+    const splitMessage = parsedMessage.split(' ').splice(1);
+    const cityName = `${splitMessage.join(' ')}`;
```

Comment:
Even better -
You don't need the variable `splitMessage` at all. Replace both lines by
```js
const cityName = parsedMessage.substr(parsedMessage.indexOf(' ') + 1);
```